src/web/core/src/main/java/org/geoserver/web/data/layergroup/LayerGroupEditPage$LayerGroupTab.html

```diff
@@ -26,6 +26,10 @@
          <input wicket:id="generateBounds" type="submit" wicket:message="value:generateBounds"/>
          <input wicket:id="generateBoundsFromCRS" type="submit" wicket:message="value:generateBoundsFromSRS"/>
        </li>
+       <li>
+         <label for="sortIndex"><wicket:message key="sortIndex">Index for sorting layers on getcapabilities</wicket:message></label>
```

Comment:
"on getcapabilities" should probably be "in GetCapabilities". @bradh I need the help of a native english speaker here, what say you? :-)

Reply:
How about:

> GetCapabilities order:

Order implies sorting?